src/main/java/Ingridient.java

```diff
@@ -0,0 +1,20 @@
+package pl.javastart.task;
+
+public class Ingridient {
+
+    String ingridientName;
```

Comment:
Jeżeli klasa nazywa się Ingredient, to nie musisz już tego dodawać do nazw pól, wystarczy samo name